CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+# Version 2.1.0
+- Introduced Heartbeat support to monitor application health periodically.
```

Comment:
rephrase to something like 'Heartbeat feature which sends periodic hearbeats with basic info about the app/runtime to Application Insights'

Reply:
I think this sentence is sufficient. I can link the PR for description.